big-num/omeganum.lua

```diff
@@ -481,7 +484,7 @@ function Big:parse(input)
                 if (intPartLen - 1 >= LONG_STRING_MIN_LENGTH) then
                     b[1] = math.log10(b[1]) + log10LongString(string.sub(a[i], 1, intPartLen - 1))
                     b[2] = 1;
-                elseif ((a[i] ~= nil) and (a[i] ~= "") and (tonumber(a[i]) ~= nil)) then
+                elseif ((a[i] ~= nil) and (a[i] ~= "")) then
```

Comment:
```suggestion
                elseif ((a[i] ~= nil) and (a[i] ~= "") and (tonumber(a[i]) ~= nil)) then
```

This needs to stay like this. Please.

Reply:
oops, don't remember removing that 
might have been a bad merge